server/services/config/config.go

```diff
@@ -50,6 +50,8 @@ type Configuration struct {
 	LocalModeSocketLocation  string            `json:"localModeSocketLocation" mapstructure:"localModeSocketLocation"`
 	EnablePublicSharedBoards bool              `json:"enablePublicSharedBoards" mapstructure:"enablePublicSharedBoards"`
 	FeatureFlags             map[string]string `json:"featureFlags" mapstructure:"featureFlags"`
+	EnableDataRetention      bool              `json:"enable_data_retention" mapstructure:"enable_data_retention"`
```

Comment:
Should this be captured for metrics?

Reply:
This is actually captured in the mattermost-server, as this is set in that conf file. However, I am going to create a card for pass config setting in daily metrics, we currently don't do that.